src/examples/example_get_storage.rs

```diff
@@ -28,27 +27,24 @@ fn main() {
     let mut api = Api::new(format!("ws://{}", url));
 
     // get some plain storage value
-    let result_str = api.get_storage("Balances", "TotalIssuance", None).unwrap();
-    let result = hexstr_to_u256(result_str).unwrap();
+    let result: u128 = api.get_storage_value("Balances", "TotalIssuance").unwrap();
     println!("[+] TotalIssuance is {}", result);
 
-    // get Alice's AccountNonce
+    // get StorageMap
     let accountid = AccountKeyring::Alice.to_account_id();
-    let result_str = api
-        .get_storage("System", "AccountNonce", Some(accountid.encode()))
+    let result: Hash = api
+        .get_storage_map("System", "BlockHash", 1u32).or(Some(Hash::default()))
         .unwrap();
-    let result = hexstr_to_u256(result_str).unwrap();
-    println!("[+] Alice's Account Nonce is {}", result.low_u32());
+    println!("[+] block hash for blocknumber 42 is {:?}", result); 
```

Comment:
Honestly, storage map key `1u32` for the `block_hash` resolves to blocknumber 42?? Nice